pixeltable/env.py

```diff
@@ -508,8 +512,25 @@ def _init_metadata(self) -> None:
         assert self._sa_engine is not None
         from pixeltable import metadata
 
-        metadata.schema.base_metadata.create_all(self._sa_engine, checkfirst=True)
-        metadata.create_system_info(self._sa_engine)
+        max_retries = 3
```

Comment:
need better solution i guess im going to setup tables during db creation commit it toss that db connection
for all subsequent db connections they should see tables and skip them also systeminfo table wont update row 

we need to make sure upgrade metadata is not happening from 2 places at the same time may be we disable for now?